lib/spoom/git.rb

```diff
@@ -14,11 +14,12 @@ def self.exec(command, *arg, path: '.')
       return "", "Error: `#{path}` is not a directory.", false unless File.directory?(path)
       opts = {}
       opts[:chdir] = path
-      _, o, e, s = Open3.popen3(*T.unsafe([command, *T.unsafe(arg), opts]))
+      i, o, e, s = Open3.popen3(*T.unsafe([command, *T.unsafe(arg), opts]))
       out = o.read.to_s
       o.close
       err = e.read.to_s
       e.close
+      i.close
```

Comment:
This was breaking locally with an error related to having too many files open from `Open3`.

Reply:
Good catch ❤️